balena-docs/custom-build.md

```diff
@@ -1,3 +1,24 @@
 # Building Your Own Image
 
+To build your own custom balenaOS image, refer to the README for respective device type repository hosted in the [balenaOS GitHub org](https://github.com/balena-os/) for more information.
+
+Identifying the right repository is important. All device type repositories will start with `balena-*`. As we build device types of similar specification or board manufacturers together inside one repository. For example, to find the right repository for the device type Raspberry Pi 3, there will be a file in the repository named in the format `machine-name.coffee`. Here are the steps to finding the right repository for instructions:
+
+1. Find the [device type slugs](https://docs.balena.io/reference/hardware/devices/) for the board you are looking for. In this case, Raspberry Pi 3's slug is `raspberrypi3`.
```

Comment:
Let's replace `device type slugs` with `Machine Name` which is the name that appears as a header in https://docs.balena.io/reference/hardware/devices/